simpledotorg/simple-server
# Dashboard: Chartjs - move all colors to a single occurrence

**Story card:** [sc-9552](https://app.shortcut.com/simpledotorg/story/9552/phase-1-dry-colors-data-objects)

## Because

Duplicate colors object in report.js

There are 2 different sections with different structures for creating the graphs inside Reports.js
Each section has its own colors object with the same key values.

## This addresses

DRY out the reports.js code
Prep towards updating to v3.9.1

## Test instructions

Ensure that `medications_dispensation` flipper flag is enabled on localhost

Check that all charts render correctly on 'reports'

Charts to check:
- BP controlled
- Registrations
- BP not controlled
- Missed visits
- Treatment status of patients under care
- Lost to follow-up
- Medications - Days of medications dispensed to follow-up patients
- Cohort report - Quarterly report

Check charts on both 
- Hypertension
- Diabetes


Check that small charts render correctly inside 'trends'
- Its the tiny little line graph with no axis.
